stable/namerd/templates/config.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}-config
+data:
+{{ if .Values.config }}
+  config.yaml: |-
+{{ toYaml .Values.config| indent 4 }}
+{{ else }}
+  config.yaml: |-
+    admin:
+      ip: 127.0.0.1
+      port: 9991
+    storage:
+      kind: io.l5d.k8s
+      experimental: true
+    namers:
+      - kind: io.l5d.k8s
+        experimental: true
```

Comment:
I was able to remove the `namers` experimental flag, but namerd refused to startup without experimental set on storage:

> `io.buoyant.namerd.NamerdConfig$$anon$1: The io.buoyant.namerd.storage.K8sConfig storage is experimental and must be explicitly enabled by setting the 'experimental' parameter to true.`